superflore/generators/ebuild/ebuild.py

```diff
@@ -123,7 +123,7 @@ def get_ebuild_text(self, distributor, license_text):
             if len(split) > 1:
                 # they did something like "BSD,GPL,blah"
                 ret += 'LICENSE="( '
-                ret += ' '.join([get_license(l) for l in split])
+                ret += ' '.join([get_license(l.lstrip()) for l in split])
```

Comment:
Why only a lstrip here? You might as well strip both ends of the tokens.

Reply:
That's a good point. Fixed.